test/unit/legacy_wrappers/gridfs.test.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const {
+  MongoClient: LegacyMongoClient,
+  GridFSBucket: LegacyGridFSBucket,
+  FindCursor: LegacyFindCursor,
+  GridFSBucketWriteStream: LegacyGridFSBucketWriteStream
+} = require('../../../src/index');
+const mongodbDriver = require('mongodb');
+const sinon = require('sinon');
+const { expect } = require('chai');
+const { toLegacy } = require('../../../src/utils');
+
+const iLoveJs = 'mongodb://iLoveJavascript';
+
+describe('legacy_wrappers/gridfs.js', () => {
+  let client;
+  let bucket;
+  beforeEach(async function () {
+    client = new LegacyMongoClient(iLoveJs);
+    bucket = new LegacyGridFSBucket(client.db());
+  });
+
+  afterEach(async function () {
+    sinon.restore();
+    await client.close();
+  });
+
+  it("should convert bucket.find's FindCursor to legacy version", () => {
```

Comment:
unexpected double quotes - lint?

Reply:
nope, there's an apostrophy in here, but I rephrased it now :)